code/modules/mob/living/silicon/robot/drone/drone_items.dm

```diff
@@ -64,7 +64,7 @@
 	..()
 	can_hold = typecacheof(can_hold)
 
-/obj/item/gripper/verb/drop_item()
+/obj/item/gripper/verb/drop_item_gripped()
```

Comment:
naming a verb the same thing as an existing proc isn't a generally good practice.